src/hotspot/share/runtime/mutex.hpp

```diff
@@ -81,8 +82,14 @@ class Mutex : public CHeapObj<mtSynchronizer> {
        native         = max_nonleaf    +   1
   };
 
+ private:
+  // The _owner field is only set by the current thread, either to itself after it has acquired
+  // the low-level _lock, or to NULL after it has released the _lock. Accesses by any thread other
```

Comment:
s/after/before/

Reply:
More specifically, since there are two "after", this is about the release of the lock.